### PR TITLE
Reduce api token logging

### PR DESCRIPTION
**Pull request by zackkrida**
## Description

We've had a pretty large increase to our log storage costs in production. This PR tries to remove some of the logs from the API token process. I'm opening to keeping a subset of these or whatever the minimum amount of information we need to debug problems with this is.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — COMMENTED** [review]
Given that the implementation currently works, I'm ok with removing the logging.

But would it be a better solution to make the `log` function a no-op on the production server-side? That allows us to keep the detailed logging in dev untouched, just in case.

https://github.com/WordPress/openverse-frontend/blob/bb12c58bae7d573e17bf9b622edb930ae54306ec/src/utils/console.ts#L3-L8

**Comment by zackkrida** [comment]
@dhruvkb I'm not sure; they also clutter local development considerably too. Perhaps I should comment them out rather than delete for easy restoration if there's ever issues with the token implementation?

**Review by dhruvkb — APPROVED** [review]
Better to delete them if they aren't useful in dev either.

File: src/plugins/api-token.server.ts [hunk 1]
```diff
@@ -176,12 +161,6 @@ const apiToken: Plugin = async (context, inject) => {
   let openverseApiToken: string | undefined
   try {
     openverseApiToken = await getApiAccessToken(context)
```
Comment (obulat) [inline src/plugins/api-token.server.ts:163]: Maybe we should leave these lines so that it's clear if we are using the token when fetching or not?

**Review by obulat — APPROVED** [review]
LGTM! This has been making the dev output almost unusable, but we might want to add at least some indication of whether the token is added to requests or not.

File: src/plugins/api-token.server.ts (hunk 1 above)
Comment (zackkrida) [reply src/plugins/api-token.server.ts:163]: I'm not sure. We're already sending the `error('Unable to retrieve API token, clearing existing token', e)` when things fail, which i think would be enough information to debug this token code in the event of a problem.

**Review by sarayourfriend — APPROVED** [review]
LGTM!

**Pull request merged**